### DISCUSSION TITLE
no WM_CREATE

### DISCUSSION THREAD

**Discussion by sjk2lq** [mhammond/pywin32#2522] [General]
Create a form with CreateWindowEx, but there is no WM_CREATE in the callback function

**Comment by kxrob** [comment]
Form? Is it a dialog class? When a DlgProc is used you get no WM_CREATE, but probably WM_INITDIALOG 

(This seems more like a forum question)

**Comment by Avasam** [comment]
If this was not a question and trying to report an issue, then please provide a minimal reproducible example (https://stackoverflow.com/help/minimal-reproducible-example) with your expectations and current behaviour.
